certified-connectors/DocuSignDemo/script.csx

```diff
@@ -1365,12 +1365,21 @@ public class Script : ScriptBase
     int recipientCount = envelope["recipients"]["recipientCount"].ToObject<int>();
     var recipientCountInNaturalLanguage = (recipientCount > 1) ?
         (" and " + (recipientCount - 1).ToString() + " others have ") : " "; 
-
+ 
     JArray documentArray = (envelope["envelopeDocuments"] as JArray) ?? new JArray();
-    var documentCountInNaturalLanguage = (documentArray.Count > 1) ?
-      (" and " + (documentArray.Count - 1).ToString() + " other documents ") : " ";
+    var documentCount = documentArray.Count;
+    string documentCountInNaturalLanguage = "";
+
+    if (documentCount == 3)
+    {
+      documentCountInNaturalLanguage = $" and 1 other document";
+    }
+    else if (documentCount > 3)
+      {
```

Comment:
Fix the spacing.
You can install prettier plugin for visual studio code and it will do auto formatting of code.